index.html

```diff
@@ -4,7 +4,7 @@
     <title>Fedora API Specification</title>
     <meta charset='utf-8'>
     <meta name="description" content="Fedora API Specification">
-    <script src='https://www.w3.org/Tools/respec/respec-w3c-common'
+    <script src='http://fedora.info/spec/respec/v14.0.13/respec-w3c-common'
```

Comment:
Please use `//fedora.info/spec...` (omitting http:), as this change may cause many browsers to _not_ load the javascript if the page is served via HTTPS. Using `src="//fedora.info...` will tell browsers to use the same protocol (HTTP or HTTPS) as is used by the main page.